test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -211,11 +212,13 @@ def test_mask_along_axis(self, shape, mask_param, mask_value, axis):
 
         assert mask_specgram.size() == specgram.size()
         assert num_masked_columns < mask_param
+        self.assertEqual(specgram, specgram_copy)
```

Comment:
the intention here is to specifically test that the original is not changed. at a minimum, i'd expect a comment justifying why this is here.  however, i would usually expect a separate test for this, so we know that this was the intention of the test. having a separate test would also make it easier to extend to other functionals if we want to.

thoughts?

Reply:
I agree with this point. Each unit test should be testing only one thing, so that when a unit test fails we know what aspect of the functionality is not met.

Can you make new test methods with short description and reference the issue number in docstring.
See examples of test descriptions like

https://github.com/pytorch/audio/blob/0c263a935f178490bda78c046e220e84b095447d/test/torchaudio_unittest/backend/sox_io/save_test.py#L381-L382

https://github.com/pytorch/audio/blob/0c263a935f178490bda78c046e220e84b095447d/test/torchaudio_unittest/backend/sox_io/load_test.py#L284-L293